src/Renderer/Shader/ShaderChunk.js

```diff
@@ -0,0 +1,17 @@
+import pitUV from './Chunk/pitUV.glsl';
```

Comment:
Could you load directly load shader itowns chunk in main?
You could rename this file in `itownsShaderChunk`

Reply:
I guess we can keep the name `ShaderChunk.js` and have client code use `itowns.ShaderChunk`.

ok to expose `ShaderChunk.js` in main, but is there a way to tag this and other lowlevel API as such, to segment the THREE aware and unaware APIs ?

Reply:
it isn't to expose `ShaderChunk.js` but to load directly all `itowns Chunk`.
With `itownsShaderChunk.js`, it's clearer to differentiate the chunks from itowns and those from THREE